diff --git a/processSliceOfValues.s b/processSliceOfValuesCopy.s
index 7bc7cc4..2524b66 100644
--- a/processSliceOfValues.s
+++ b/processSliceOfValuesCopy.s
@@ -1,4 +1,4 @@
-main.processSliceOfValues STEXT nosplit size=93 args=0x18 locals=0x0 funcid=0x0 align=0x0
+main.processSliceOfValuesCopy STEXT nosplit size=102 args=0x18 locals=0x0 funcid=0x0 align=0x0
 TEXT
 MOVQ
 FUNCDATA
@@ -6,34 +6,37 @@ FUNCDATA
 FUNCDATA
 FUNCDATA
 PCDATA
+MOVQ
 XORL
 JMP
 MOVQ
-SHLQ
 MOVQ
 MOVQ
+SHLQ
 LEAQ
 LEAQ
 MOVQ
 MOVQ
 MOVQ
 IMULQ
-ADDQ
+LEAQ
 SARQ
 SARQ
 SUBQ
-ADDQ
-INCQ
+LEAQ
+LEAQ
 NOP
 MOVQ
 MOVQ
-MOVQ
+LEAQ
+NOP
 CMPQ
 JGT
 RET
-0x0000 48 89 44 24 08 31 c9 eb 4e 48 89 ca 48 c1 e2 04  H.D$.1..NH..H...
-0x0010 48 8b 34 10 48 8b 7c 10 08 48 8d 34 76 48 8d b6  H.4.H.|..H.4vH..
-0x0020 d2 04 00 00 49 89 c0 48 b8 ab aa aa aa aa aa aa  ....I..H........
-0x0030 aa 49 89 d1 48 f7 ef 48 01 fa 48 d1 fa 48 c1 ff  .I..H..H..H..H..
-0x0040 3f 48 29 fa 48 83 c2 e7 48 ff c1 4b 89 34 08 4b  ?H).H...H..K.4.K
-0x0050 89 54 08 08 4c 89 c0 48 39 cb 7f ad c3           .T..L..H9....
+0x0000 48 89 44 24 08 48 89 c1 31 d2 eb 54 48 8b 70 08  H.D$.H..1..TH.p.
+0x0010 48 8b 38 49 89 d0 49 c1 e0 04 48 8d 3c 7f 48 8d  H.8I..I...H.<.H.
+0x0020 bf d2 04 00 00 49 89 c1 48 b8 ab aa aa aa aa aa  .....I..H.......
+0x0030 aa aa 49 89 d2 48 f7 ee 4c 8d 1c 32 49 d1 fb 48  ..I..H..L..2I..H
+0x0040 c1 fe 3f 49 29 f3 49 8d 73 e7 49 8d 52 01 4a 89  ..?I).I.s.I.R.J.
+0x0050 3c 01 4a 89 74 01 08 49 8d 41 10 0f 1f 44 00 00  <.J.t..I.A...D..
+0x0060 48 39 d3 7f a7 c3                                H9....
