diff --git a/processSliceOfValues.s b/processSliceOfPointers.s
index 7bc7cc4..704fe40 100644
--- a/processSliceOfValues.s
+++ b/processSliceOfPointers.s
@@ -1,4 +1,4 @@
-main.processSliceOfValues STEXT nosplit size=93 args=0x18 locals=0x0 funcid=0x0 align=0x0
+main.processSliceOfPointers STEXT nosplit size=86 args=0x18 locals=0x0 funcid=0x0 align=0x0
 TEXT
 MOVQ
 FUNCDATA
@@ -9,7 +9,6 @@ PCDATA
 XORL
 JMP
 MOVQ
-SHLQ
 MOVQ
 MOVQ
 LEAQ
@@ -31,9 +30,9 @@ MOVQ
 CMPQ
 JGT
 RET
-0x0000 48 89 44 24 08 31 c9 eb 4e 48 89 ca 48 c1 e2 04  H.D$.1..NH..H...
-0x0010 48 8b 34 10 48 8b 7c 10 08 48 8d 34 76 48 8d b6  H.4.H.|..H.4vH..
-0x0020 d2 04 00 00 49 89 c0 48 b8 ab aa aa aa aa aa aa  ....I..H........
-0x0030 aa 49 89 d1 48 f7 ef 48 01 fa 48 d1 fa 48 c1 ff  .I..H..H..H..H..
-0x0040 3f 48 29 fa 48 83 c2 e7 48 ff c1 4b 89 34 08 4b  ?H).H...H..K.4.K
-0x0050 89 54 08 08 4c 89 c0 48 39 cb 7f ad c3           .T..L..H9....
+0x0000 48 89 44 24 08 31 c9 eb 47 48 8b 14 c8 48 8b 32  H.D$.1..GH...H.2
+0x0010 48 8b 7a 08 48 8d 34 76 48 8d b6 d2 04 00 00 49  H.z.H.4vH......I
+0x0020 89 c0 48 b8 ab aa aa aa aa aa aa aa 49 89 d1 48  ..H.........I..H
+0x0030 f7 ef 48 01 fa 48 d1 fa 48 c1 ff 3f 48 29 fa 48  ..H..H..H..?H).H
+0x0040 83 c2 e7 48 ff c1 49 89 31 49 89 51 08 4c 89 c0  ...H..I.1I.Q.L..
+0x0050 48 39 cb 7f b4 c3                                H9....
